pvacseq/pvacseq.py

```diff
@@ -35,6 +35,13 @@ def main():
     )
     coverage_filter_parser.set_defaults(func=lib.coverage_filter)
 
+    generate_protein_fasta_parser = subparsers.add_parser(
+        "generate_protein_fasta",
+        help="Generate a fasta file from a VCF with protein sequences of mutations and matching wildtypes",
```

Comment:
What would the `annotated` hear mean? That the header encodes certain information?